cli/src/cmd/forge/test/mod.rs

```diff
@@ -80,6 +80,10 @@ pub struct TestArgs {
     #[clap(long, short, help_heading = "Display options")]
     json: bool,
 
+    /// Exit tests after first failure
+    #[clap(long, help = "Stops running tests after the first failure")]
+    pub fail_fast: bool,
```

Comment:
I think we also separately need to add config file support so you can use `fail_fast` in the `foundry.toml`? Also does this apply to all tests (regular, fuzz, and invariant)?

Reply:
Tested fuzz and invariant in comment below, and will figure out the config file support now!